features/ubuntu_pro.feature

```diff
@@ -0,0 +1,24 @@
+@uses.config.machine_type.pro.aws
+Feature: Command behaviour when attached to an UA subscription
+
+    @series.xenial
+    @series.bionic
+    @series.focal
+    Scenario Outline: Attached refresh in a trusty machine
```

Comment:
I am unable to run this scenario. Right now, when running it, I receive the following error:

```
in before_scenario: ClientError: An error occurred (InvalidParameterValue) when calling the CreateSubnet operation: invalid value for parameter ipv6-cidr-block: 64
```

I have checked to see if I am using the right `pycloudlib` branch, but the problem still persists

Reply:
Also, these are the version of my boto libs:

```
boto3==1.14.20
botocore==1.17.20
```

Reply:
I think something in your specific ec2 configuration opts to avoid ipv6 setup on subnets. I bet I have configured that on my ec2 account by default so we might have to fix pycloudlib to handle non-ipv6 setups.

Can you run this script in your pycloudlib repo root
tox -e pylint
.tox/pylint/bin/python3 ./1.py
https://paste.ubuntu.com/p/Jt4G9C9Ddb/

Reply:
Ok this is a timing issue when creating a subnet before the vpc is ready (and includes the amazon allocated IPv6 address).
I've added a vpc.wait_until_available and vpc.reload() before creating subnets to avoid this issue. https://github.com/canonical/pycloudlib/pull/8